boolean/boolean.py

```diff
@@ -1175,7 +1175,11 @@ def start_operation(ast, operation):
         elif token == TOKEN_RPAR:
             while True:
                 if ast[0] is None:
-                    raise TypeError('Bad closing parenthesis at line: %(row)d, column: %(col)d' % locals())
+                    if isinstance(position, tuple):
+                        row, col = position
+                        raise TypeError('Bad closing parenthesis at line: %(row)d, column: %(col)d' % locals())
+                    else:
+                        raise TypeError('Bad closing parenthesis at position: %(position)d' % locals())
```

Comment:
Absolutely, I'd prefer that. I thought about doing it that way first, but then I changed it so the message remained as it originally was for row and line positions.